### ISSUE TITLE
"Not In Empty" handling is broken on PostgreSQL

### ISSUE THREAD

**Issue by vito** [Masterminds/squirrel#101]
It generates a query like...:

```sql
DELETE FROM worker_base_resource_types WHERE worker_name = $1 AND (1)
```

Which fails with:

```
<*pq.Error | 0xc4203dcd80>: {
    Severity: "ERROR",
    Code: "42804",
    Message: "argument of AND must be type boolean, not type integer",
    Detail: "",
    Hint: "",
    Position: "68",
    InternalPosition: "",
    InternalQuery: "",
    Where: "",
    Schema: "",
    Table: "",
    Column: "",
    DataTypeName: "",
    Constraint: "",
    File: "parse_coerce.c",
    Line: "1082",
    Routine: "coerce_to_boolean",
}

```

**Comment by lann** [comment]
Revered c199e0953881c25d1266988c4c3b4a1d6c005046 to fix this. I'm going to look into DB integration tests in Travis to prevent this in the future.

**Comment by lann** [comment]
FYI, integration tests added in 09b59954b5cfb11d5957ca38c79de69d980c2c2a; should cut down on database-specific breakage like this.

**Comment by lann** [comment]
I pushed a new fix to #99 (a6b93000bd219143c56c16e6cb1c4b91da3f224b). There is now a regression test that should prevent your failure but be aware that since you were hitting this condition you may be accidentally depending on the old buggy behavior.
